sass/base/components/media-block/_media-block.scss

```diff
@@ -29,7 +30,7 @@
       @include opacity(1);
     }
     #play-icon {
-      transform: scale(1.2);
+      transform: scale(1.1667);
```

Comment:
Why such a tiny change? And does the long decimal value work okay for older browsers still?

Reply:
The reason for this is that the icon should be 70px on hover. I've tried not using scale and adding the transition on the width, but the transition wasn't as smooth. Will run this by Leigh tomorrow.